dev/io.openliberty.jakarta.jsonp.2.1_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -133,9 +126,6 @@
 							<forkCount>1</forkCount>
 		                    <reuseForks>false</reuseForks>
 							<dependenciesToScan>jakarta.json:jakarta.json-tck-tests-pluggability</dependenciesToScan>
-							<systemPropertyVariables>
-								<jimage.dir>${project.build.directory}/jdk11-bundle</jimage.dir>
-							</systemPropertyVariables>
```

Comment:
So it looks like this change was part of another PR you were working on.  I'm guessing you created a new branch off of that branch.  Consider rebasing this branch onto `origin/release` to avoid future merge conflicts.  Let me know if you want me to show you how to do that.